Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -222,10 +222,57 @@ protected override void OnCreate(Bundle savedInstanceState)
 			base.OnCreate(savedInstanceState);
 
 			Profile.FramePartition("SetSupportActionBar");
-			AToolbar bar;
+			AToolbar bar = null;
+
+#if __ANDROID_29__
+			if (ToolbarResource == 0)
```

Comment:
If you're using API 29 and these Ids are zero then just set them using the internal layout ids.